.circleci/config.yml

```diff
@@ -147,7 +147,7 @@ jobs:
       - run: make docker.prebuilt
       - run: bin/testEnvRootMinikube.sh wait
       - run: docker images
-      - run: $GOPATH/bin/pilot-integration-test --logtostderr -hub $HUB -tag $TAG -mixer=true -auth=enable -errorlogsdir=/home/circleci/logs -use-initializer
+      - run: $GOPATH/bin/pilot-integration-test --logtostderr -hub $HUB -tag $TAG -mixer=true -auth=enable -errorlogsdir=/home/circleci/logs -use-sidecar-injection=false

```

Comment:
quick nit: can we move this entire thing into the makefile as a target? This way, you don't have to touch circleci file at all. Like have a ci-e2e-pilot target in the makefile (I think HUB/TAG are already there). You can move mixer/auth enable/errorlogsdir into it as well.

Reply:
https://github.com/istio/istio/issues/2807